index.html

```diff
@@ -470,7 +471,8 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
       Represents a <a>Thing Description</a> (<a>TD</a>) as
       <a href="https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/">defined</a> in
       [[!WOT-TD]]. It is expected to be
-      a <a href="https://infra.spec.whatwg.org/#parse-json-bytes-to-a-javascript-value">parsed JSON object</a> that is validated using <a href="https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/#json-schema-for-validation">JSON Schema validation</a>.
+	  a <a href="https://infra.spec.whatwg.org/#parse-json-bytes-to-a-javascript-value">parsed JSON object</a> that is validated using  <a href="https://www.w3.org/TR/2020/WD-wot-thing-description11-20201124/#json-schema-for-validation">JSON Schema validation</a>.
+	  <a>Thing Descriptions</a> are encoded in a JSON format that also allows <a>JSON-LD</a> processing.
```

Comment:
Not sure about the phrasing here 🤔 . May I suggest something like this:
> <a>Thing Descriptions</a> are encoded in a JSON format that also might contain JSON-LD keywords and features.

In principle, any JSON allows JSON-LD processing, so the previous sentence does not help too much.

Reply:
Taken from https://w3c.github.io/wot-thing-description/#abstract.
Moreover, I am not sure if _any_ JSON allows proper JSON-LD processing.

@relu91 do you have an alternative proposal?

Reply:
I see, tbh I don't have a better wording besides the one that I already gave above 😢 . 

> Moreover, I am not sure if any JSON allows proper JSON-LD processing.

Yeah that's also true this JSON is an example:
```json
{
  "@context": 1
}
```

However, it still feels a little bit vague.

Reply:
@zolkis any opinion?
Can we live with the vague statement taken from the TD spec or shall we improve?

Reply:
Scripting API 2022-01-31

Remove JSON-LD as a whole.
@danielpeintner will update PR.

Merged afterwards.

Reply:
resolved by https://github.com/w3c/wot-scripting-api/pull/368/commits/9acc6d799f5d72020ab8a5f892a39eedc1bbc51c